### ISSUE TITLE
compatibility with node: console.dir

### ISSUE THREAD

**Issue by janwilmake** [oven-sh/bun#4429]
### What version of Bun is running?

0.8.1

### What platform is your computer?

Darwin 21.6.0 arm64 arm

### What steps can reproduce the bug?

I think this difference should be documented somewhere as it's not the same as Node.js

This is what I encountered:

`console.dir([/*long array*/], { maxArrayLength: 999 })`

it doesn't care about the options it seems, it just prints them.

In node.js this was typesafe and did wat you asked it to. In bun, it seems to adopt something else and just prints the last argument

This is what I was expecting:

https://techozu.com/show-all-items-console-log-node/

### What is the expected behavior?

It parses the options and doesn't print them

### What do you see instead?

It prints the options as log

**Comment by zoodogood** [comment]
Bun v1.0.0
`console.dir(logs, {showHidden: true, depth: 0})` ignores the second argument as a options set and outputs it into the console as an object

**Comment by bompus** [comment]
I came here with the same issue. For now, I am using this workaround:

```
import { inspect } from 'node:util';
const obj = { a: { b: 1 } };
console.log(inspect({ obj }, { colors: true, depth: 1 }));
```

**Comment by Jarred-Sumner** [comment]
Fixed between Bun v1.0.0 and Bun v1.1.0
